index.js

```diff
@@ -4,18 +4,30 @@ const { getApolloServer } = require("./apollo");
 
 (async () => {
   const app = express();
-  app.use(express.json({ limit: "2mb" }));
+  const jsonBodyLimit = getENV("GRAPHQL_JSON_BODY_LIMIT", "2mb");
+  app.use(express.json({ limit: jsonBodyLimit }));
 
   const server = await getApolloServer();
   server.applyMiddleware({ app });
```

Comment:
I'm not sure if we want to apply middleware each time it's probably pushing new middleware to stack and not removing the previous one. This basically creates dynamic middleware. The same would apply to lambda handler as well. Any thoughs about this?

1st part of update
```suggestion
    let middleware = server.getMiddleware({});

  app.use((req, res, next) => {
    middleware(req, res, next);
  });
```

Reply:
I tried your solution, and it works better I think. So I implemented that instead. The schema repository idea sounds good too! I would love to get noticed when it's ready to test out